Lonely/Lonely/GameLib/DirectX/DirectX.cpp

```diff
@@ -0,0 +1,199 @@
+/**
+* @file	Direct3D.h
```

Comment:
```suggestion
* @file	Direct3D.cpp
```
ですね。このままだと頑張ってコメント書いてもdoxygenで出力されないやつ。
あるあるです。

Reply:
ファイル名間違っているので再指摘しておきました。